docs/elements.md

```diff
@@ -411,21 +411,21 @@ def generate(data):
+| ---------------- | ----------------------- | ---------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `answers-name`   | string                  | —          | Variable name to store data in. Note that this attribute has to be unique within a question, i.e., no value for this attribute should be repeated within a question. |
+| `weight`         | integer                 | 1          | Weight to use when computing a weighted average score over elements.                                                                                                 |
+| `correct-answer` | string                  | special    | Correct answer for grading. Defaults to `data["correct_answers"][answers-name]`. If `base` is provided, then this answer must be given in the provided base.         |
+| `allow-blank`    | boolean                 | false      | Whether or not an empty input box is allowed. By default, empty input boxes will not be graded (invalid format).                                                     |
+| `blank-value`    | integer                 | 0 (zero)   | Value to be used as an answer if element is left blank. Only applied if `allow-blank` is `"true"`.                                                                   |
+| `label`          | text                    | —          | A prefix to display before the input box (e.g., `label="$x =$"`).                                                                                                    |
+| `suffix`         | text                    | —          | A suffix to display after the input box (e.g., `suffix="items"`).                                                                                                    |
+| `base`           | integer                 | 10         | The base used to parse and represent the answer, or the special value 0 (see below).                                                                                 |
+| `display`        | `"block"` or `"inline"` | `"inline"` | How to display the input field.                                                                                                                                      |
+| `size`           | integer                 | 35         | Size of the input box.                                                                                                                                               |
+| `show-help-text` | boolean                 | true       | Show the question mark at the end of the input displaying required input parameters.                                                                                 |
+| `placeholder`    | string                  | -          | Custom placeholder text. If not set, defaults to `integer` if `base` is 10, otherwise `integer` in base `base`".                                                     |
```

Comment:
I had a bit of uncertainty for some values in the description column, would we want to edit it so that it's `"integer"` or `integer` in this case?